app/controllers/v0/profile/connected_applications_controller.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module V0
+  module Profile
+    class ConnectedApplicationsController < ApplicationController
+      def index
+        render json: { data: grants_by_app.values }
+      end
+
+      def destroy
+        app = grants_by_app(false)[connected_accounts_params[:id]]
+        app[:attributes][:grants].each do |grant|
+          delete_response = okta_service.delete_grant(@current_user.uuid, grant[:id])
+          unless delete_response.success?
+            log_message_to_sentry("Error deleting grant #{connected_accounts_params[:id]}", :error,
+                                  body: delete_response.body)
+            raise 'Unable to delete grant'
+          end
+        end
+
+        head :no_content
+      end
+
+      private
+
+      def okta_service
+        @okta_service ||= Okta::Service.new
+      end
+
+      def get_logo(app_id)
+        app_response = okta_service.app(app_id)
+        if app_response.success?
+          app_response.body['_links']['logo'].last['href']
+        else
+          log_message_to_sentry('Error fetching app', :error,
+                                body: app_response.body)
+          raise 'Unable to fetch app'
```

Comment:
I think it'd be better to recover from errors here, since being unable to display a logo shouldn't cause a fatal error in the application.

Reply:
It should still log to sentry though.

Reply:
Sounds good, I'll update